exercises/rna-transcription/Sources/RnaTranscriptionExample.swift

```diff
@@ -10,9 +15,12 @@ struct Nucleotide {
 
     private let dnaToRna: [Character:String] = [ "G": "C", "C": "G", "T": "A", "A": "U" ]
 
-    private func transcribe(_ dict: [Character : String]) -> String {
+    private func transcribe(_ dict: [Character : String]) throws -> String {
         var tempText = ""
         for each in self.value.characters {
+            if (dict[each] == nil) {
```

Comment:
The linter is complaining here. Run the autocorrect to see if some of the others go away.